### DISCUSSION TITLE
How to change Graphics to Sprite

### DISCUSSION THREAD

**Discussion by vsTianhao** [pixijs/pixijs#10534] [Q&A]
I've looked up a lot of information that says to `renderer.generateTexture()`，Not sure where the `renderer` came from, though.
`app.renderer` is undefined, and `autoDetectRenderer` need params

**Comment by bigtimebuddy** [comment]
Here's a simple example:
https://pixiplayground.com/#/edit/oHql2XzLl86yVWkretZ19
